pipelines/auth/pip.yaml

```diff
@@ -0,0 +1,53 @@
+name: Configure pip to fetch Chainguard Libraries dependencies.
+
+needs:
+  packages:
+    - busybox
+    - curl
+    - jq
+
+inputs:
+  identity:
+    description: the identity to use for Chainguard Libraries
+    default: "720909c9f5279097d847ad02a2f24ba8f59de36a/a49c7fedc33adf69"
+  group:
+    description: the group to use for Chainguard Libraries
+    default: "720909c9f5279097d847ad02a2f24ba8f59de36a"
+  guarded:
```

Comment:
how about "no-index-fallback" or "guarded-only"?

Reply:
just the name. The default should be to *only* install guarded things. I like names that i can read and I immediately understand broadly what it does. `guarded` true/false doesn't meet that bar for me. We're installing guarded libs either way. "only-guarded" or "no-index-fallback" or something like that tells me what it does on the tin.

TBH, I'm actually not even sure I like having a way to enable a fallback. I'm sure we'll need an escape hatch at times, but I'd prefer something more granular and declarative than "whatever the resolver happens to choose at that moment". I'd like for developers to explicitly list the things that they are pulling from any other index and why.

WDYT about putting this on hold? Or making it a flag that is clearly only meant for debug?